modules/livestore/partition_reader.go

```diff
@@ -169,6 +176,12 @@ func collectFetchErrs(fetches kgo.Fetches) (_ error) {
 
 func (r *PartitionReader) consumeFetches(ctx context.Context, fetches kgo.Fetches) *kadm.Offset {
 	// Pass offset and byte information to the live-store
+	partLabel := strconv.Itoa(int(r.partitionID))
```

Comment:
I hate the fact we have to do this, but changing it is relatively painful and holding a string for the id in addition also feels bad.